stable/kubernetes-dashboard/values.yaml

```diff
@@ -80,6 +82,7 @@ ingress:
 
   ## Kubernetes Dashboard Ingress path
   ##
+  ## For GCE ingress the path should be /*
   path: /
```

Comment:
Honestly, I do not understand how it is working right now because when I only set `/` as the path, only the request on `/` route are routed to the dashboard, the rest is routed to default backend. I think /* is required but I'm not in position to test that on nginx ingress.

Reply:
Oh I found the answer, nginx ingress and gce ingress doesn't behave equally with paths :(